x-pack/plugin/sql/qa/server/src/main/resources/arithmetic.sql-spec

```diff
@@ -244,9 +244,9 @@ SELECT emp_no FROM test_emp WHERE 1./-salary  > 1/-6E4;
 chainedNegationDenominator
```

Comment:
Just curious: are the changes in the spec meant to make more sense of the queries, or are there test-needs?

Reply:
Just test needs. Now, only one field is allowed inside arithmetic operations of any kind.

Reply:
Right, the very scope of the PR. Thanks.